### ISSUE TITLE
Change Notice component to be in-line with Photon spec and check its colours are correct.

### ISSUE THREAD

**Issue by tofumatt** [mozilla/addons-frontend#4549]
As mentioned in #4548, it seems:

1. Our `Notice` component needs style tweaks to be in-line with Photon
2. The Photon styles are wrong re: "generic" colours for buttons (they **say** blue but **look** grey)
3. We should allow `Notice` components to more easily render buttons that are links
4. The fix for #4548 should be changed to be a button link to "Download Firefox"

**Comment by kumar303** [comment]
I asked for clarification in https://github.com/FirefoxUX/photon/issues/287 . I'm pretty sure they should be gray.

**Comment by kumar303** [comment]
I'm like 99% sure these buttons should be gray so I'm making them gray. The blue button is too much of a call to action on the page.

**Comment by ioanarusiczki** [comment]
@kumar303 I checked on AMO dev (Win10) for the filter but I do not see it displayed as shown here https://github.com/mozilla/addons-frontend/pull/7146

AMO dev 

![filter](https://user-images.githubusercontent.com/33448286/49881161-5b678480-fe36-11e8-8ef2-9fabdd208dbe.png)

**Comment by ioanarusiczki** [comment]
@kumar303 Ok, I figured out how the button becomes visible. The color is grey and it can be clicked.
It could be a darker grey (in my opinion) but while hovering it changes it's contrast.

![grey button](https://user-images.githubusercontent.com/33448286/49931780-4a6b5180-fed0-11e8-8102-97437b1a6f26.gif)

The Download Firefox button mentioned [here ](https://github.com/mozilla/addons-frontend/issues/4549#issue-304860610) is green but it's been like that for a while and the message from https://github.com/mozilla/addons-frontend/pull/4548 was removed (that was for other browsers)
